contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -351,7 +361,7 @@ func (s *SchedulerServer) serveFrameworkArtifactWithFilename(path string, filena
 }
 
 func (s *SchedulerServer) prepareExecutorInfo(hks hyperkube.Interface) (*mesos.ExecutorInfo, error) {
-	ci := &mesos.CommandInfo{
+		ci := &mesos.CommandInfo{
```

Comment:
@MikeSpreitzer I just added the hooks - thanks!   Looking at the kubernetes master tree, the code was not correctly formatted:  https://github.com/kubernetes/kubernetes/blob/master/contrib/mesos/pkg/scheduler/service/service.go -- line 354, so the patch is ok.